resources/public/api/conf/2.0/scenarios/submitBusEOPS_scenarios.md

```diff
@@ -67,5 +67,9 @@
             <td><p>BVR_UK_PROPERTY_MULTIPLE_ERRORS</p></td>
```

Comment:
I don't quite understand the BVR changes but it looks like there's a new `BUSINESS_VALIDATION_FAILURE` Gov Test Scenario which we need to add here right?